test/In.ProjectEKA.HipServiceTest/Discovery/FilterTest.cs

```diff
@@ -1,100 +1,168 @@
 namespace In.ProjectEKA.HipServiceTest.Discovery
 {
-    using System;
+    using System;
     using System.Linq;
-    using Bogus;
     using Builder;
     using FluentAssertions;
     using HipLibrary.Patient.Model;
-    using HipService.Discovery;
-    using Xunit;
+    using HipService.Discovery;
+    using Xunit;
     using static Builder.TestBuilders;
 
+
     [Collection("Patient Filter Tests")]
     public class FilterTest
     {
+        private class Linda
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5933330)